.github/dependabot.yml

```diff
@@ -34,3 +34,8 @@ updates:
     # Always increase the minimum version requirement to match the new version.
     # https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#versioning-strategy
     versioning-strategy: "increase"
+    groups:
+      nivo-deps:
```

Comment:
Is there a doc where I can learn more about this dependabot feature?

Reply:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups :)